apps/docs/app/ui/components/overview/Overview.tsx

```diff
@@ -65,7 +62,7 @@ const Overview = () => {
                     interactive
                     className="hd-component-overview-category__title"
                 >{category}</Title>
-                <HopperProvider colorScheme={theme} >
+                <HopperProvider colorScheme={theme}>
                     <div className="hd-component-overview">
                         {allComponents.filter(component =>
```

Comment:
related to the comment below:
```suggestion
                        {allComponents.filter((component: OverviewTileProps) =>
```

Reply:
this is not needed i'm pretty sure